ritm_annotation/core/training/batch_processor.py

```
@@ -0,0 +1,244 @@
+        batch: Dict[str, torch.Tensor],
+        prev_output: torch.Tensor,
+        click_idx: int
+    ) -> torch.Tensor:
+        """
+        Simulate next clicks based on current predictions.
+
+        Uses distance transform to place clicks at hard locations.
+        """
+        from ...model.is_model import get_next_points
+
+        batch_size = batch['images'].shape[0]
```

Ruff (0.14.3)

Local variable `batch_size` is assigned to but never used

Remove assignment to unused variable `batch_size`

(F841)